frontend/src/components/CreateCollectionModal/components/Content/index.tsx

```diff
@@ -144,6 +144,7 @@ const Content: FC<Props> = (props) => {
             disabled={!isValid}
             onClick={onSubmit}
             loading={isLoading}
+            data-test-id="create button"
```

Comment:
Ah actually can we make this kebab case for consistency with other `data-test-id` please? Similar to how `id` attribute is kebab case too 👌 Thank you!